core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncodingService.java

```diff
@@ -32,8 +33,13 @@
 
 /**
  * Decorates a {@link Service} to apply HTTP encoding (e.g., gzip) to an {@link HttpService}.
- * HTTP encoding will be applied if the client specifies it, the response content type is a reasonable
- * type to encode, and the response either has no fixed content length or the length is larger than 1KB.
+ * HTTP encoding will be applied if:
+ * <ul>
+ *     <li>the client specifies it</li>
+ *     <li>the response content type is a reasonable type to encode</li>
```

Comment:
I feel a little vague when I read the `reasonable` here, so how about
the response content type is one of the encodable types?